portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,6 +27,6 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    response.getWriter().println("<h1>Hello Andrés!</h1>");
```

Comment:
Can you update the above comment too to clarify what this servlet is now returning? Also get in the habit of having TODOs written as 'TODO(aabundis)' - it doesn't matter as much here but when the three of you are writing code together as a pod, it'll be useful to be able to see who is supposed to be doing/finishing what at a glance.

Reply:
Done!